_includes/footer.html

```diff
@@ -0,0 +1,24 @@
+<footer>
+  <div>
+    <h1>
+      <a href="mailto:rubyuruguay@ruby.uy">rubyuruguay@ruby.uy</a>
+    </h1>
+    <h2>
+      <a href="tel:+59898455500">+598 984 555 00</a>
+    </h2>
+    <ul>
+      <li>
+        <a href="https://twitter.com/rubymontevideo"><img src="assets/images/twitter.svg" alt="Twitter Icon"></a>
+      </li>
+      <li>
+        <a href="https://github.com/rubyuy/ruby.uy"><img src="assets/images/github.svg" alt="Github Icon"></a>
+      </li>
+      <li>
+        <a href="https://www.instagram.com/ruby_uruguay/"><img src="assets/images/instagram.svg" alt="Instagram Icon"></a>
+      </li>
+    </ul>
+  </div>
+  <div>
+    <img src="assets/images/thank_you.svg" aria-hidden="true">
```

Comment:
Use absolute paths so the includes don't get 404s when used on deeply nested routes.

```suggestion
        <a href="https://twitter.com/rubymontevideo"><img src="/assets/images/twitter.svg" alt="Twitter Icon"></a>
      </li>
      <li>
        <a href="https://github.com/rubyuy/ruby.uy"><img src="/assets/images/github.svg" alt="Github Icon"></a>
      </li>
      <li>
        <a href="https://www.instagram.com/ruby_uruguay/"><img src="/assets/images/instagram.svg" alt="Instagram Icon"></a>
      </li>
    </ul>
  </div>
  <div>
    <img src="/assets/images/thank_you.svg" aria-hidden="true">
```

Reply:
Yes, I forgot that, thanks for caching it!